### PR TITLE
Feature/sentry

### PR DESCRIPTION
**Pull request by sky1045**
### Description

1. replace mixpanel to sentry
2. make sentry transaction to track some actions execution time
3. didn't remove mixpanel analyzer yet
4. cleanup mixpanel usage
5. remove mixpanel package

### Related Links
https://sentry.io/organizations/planetariumhq/projects/nine-chronicles/?project=4503979492245504

### CODE REVIEW

**Review by boscohyun — COMMENTED** [review]
Is there any intention to create a new Tracer instead of Analyzer?

Analyzer is written to handle multiple analysis tool SDKs at the same time, so it seems enough to change mixpanel to sentrysdk.

Also, MixpanelValueFactory creates values that we want to be included in all events by default. Are these values no longer needed?

File: nekoyume/Assets/_Scripts/Analyzer.cs [hunk 1]
```diff
@@ -24,56 +22,69 @@ public class Analyzer
...
+                // Global scope: always tag for every transaction
+                scope.User = new User()
+                {
+                    Id = uniqueId
+                };
+                scope.SetTag("client-host", clientHost);
+                scope.SetTag("client-hash", clientHash);
+                scope.SetTag("target-network", targetNetwork);
+                scope.SetTag("rpc-server-host", rpcServerHost);
+                scope.SetTag("AgentAddress", agentAddress);
```
Comment (boscohyun) [inline nekoyume/Assets/_Scripts/Analyzer.cs:42]: Is it OK to remove the "AgentAddress" tag which is duplicated with User.Id?

**Review by boscohyun — APPROVED** [review]

**Comment by sky1045** [comment]
@boscohyun 
as we decided to use both mixpanel and sentry for a while, i have to rollback to preserve the codes for mixpanel not changed.
so i'll close this PR and be back with another one.

**Comment by boscohyun** [comment]
@sky1045 OK thanks!

**Pull request closed without merging**